docs/guides.md

```diff
@@ -523,3 +523,89 @@ Now we enable autostart at the virtual machine level.
 5. Checking the output 
 `# xe vm-param-list uuid=<VM_UUID> | grep other-config`
 
+## Guest UEFI Secure Boot
+
+Enabling UEFI Secure Boot for guests ensures that XCP-ng will ensure that guest VMs only execute trusted binaries.  In practice, these are the binaries released by the operating system (OS) team for the OS in running in the VM (Microsoft Windows, Debian, Alpine, etc.).
+
+### Requirements
+
+* A UEFI guest VM.
+
+### Windows
+
+The steps in this guide are all that are necessary to enable secure boot.  Just follow the instructions provided here.
```

Comment:
The Citrix docs mention that the VM must have at least two vCPUs.

Reply:
This must just be a Windows VM recommendation...  I currently have a Windows VM with one vCPU working fine with secure boot on.  It _is_ slow.  We can toss that in there though.